.gitignore

```diff
@@ -291,6 +291,7 @@ __pycache__/
 env/
 env27/
 env36/
+env310/
```

Comment:
nit, can we add a generalized pattern to ignore virtual env directories (of this form) so we don't have to add specific python version variations over time